### ISSUE TITLE
Training YOLOv5 for segmentation

### ISSUE THREAD

**Issue by himansh1314** [ultralytics/yolov5#10559]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

HI, good work with the repository. However, when I'm trying to train YOLOv5 for a segmentation task, I'm not getting good results. I have verified the data and it looks good from my end, however when I look at the data prepared by dataloader which is stored in runs/train-seg/exp/train_batch0.jpg, train_batch1.jpg, train_batch2.jpg, the labels are incorrectly placed on the image and I believe that this is the cause for bad results. I also disabled the augmentations, changed the batch size to 1, but same issue persisted. Please look into this, thanks.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@himansh1314 train_*.jpg labels are plotted correctly. If you see them incorrectly then your labels are incorrect. This is the whole purpose of providing users these images, to visually verify their annotations.
